src/request/src/middleware/src/storage/index.js

```diff
@@ -66,7 +66,7 @@ export default class Storage {
 
   findById(collection, id) {
     if (isString(id) === false) {
-      return Promise.reject(new Error('id must be a string', id));
+      return Promise.reject(new KinveyError('id must be a string', id));
```

Comment:
What is the "debug" property of the KinveyError object used for? Why add it here and but remove it on line 131?